Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -2069,7 +2057,10 @@ void gotoConfirmToPrint() {
   }
 
   #if HAS_MESH
-    void saveMesh() { TERN(AUTO_BED_LEVELING_UBL, ublMeshSave(), writeEEPROM()); }
+    void saveMesh() {
+      TERN_(MESH_BED_LEVELING, manualMeshSave();)
```

Comment:
Keep semicolons at the end of lines so that linters won't bark at us. If there is some situation where the presence of a semicolon would be bad (e.g., inside of a `switch` block) then use `OPTCODE` instead of `TERN_` and the semicolon will be added automagically.